webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -231,7 +229,10 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
             resetPagination();
             const loadableFormset = formStore.formset.get();
             Loadable.forEach(loadableFormset, (formSet) =>
-              updateSettings({ filterset: JSON.stringify(formSet) }),
+              updateSettings({
+                filterset: JSON.stringify(formSet),
+                selection: DEFAULT_SELECTION,
```

Comment:
qq: why do we wanna use ` selection: DEFAULT_SELECTION`?

Reply:
to reset selection. calling `handleSelectionChange` caused problems when it was needed as a dependency for the useEffect with `formStore.asJsonString.subscribe`.

Reply:
looks like selection across reloads, even with filters

Reply:
optional, but thinking about this piece of code again -- shouldn't setting the selection to default selection be a part of `resetPagination`?